x/feesplit/client/cli/query.go

```diff
@@ -70,15 +71,15 @@ func GetCmdQueryFees() *cobra.Command {
 	return cmd
 }
 
-// GetCmdQueryFee implements a command to return a registered contract
-// for fee distribution
-func GetCmdQueryFee() *cobra.Command {
+// GetCmdQueryFee implements a command to return a registered contract for fee
+// distribution
+func GetCmdQueryFeeSplit() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "fee-info [address]",
+		Use:     "fee-split [contract-address]",
```

Comment:
I agree  with @MalteHerrmann that this should be `evmosd q feesplit contract [contract-address]`